packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -204,13 +236,100 @@ export const styles = css`
   :host .footer {
     display: flex;
     justify-content: flex-end;
+    padding: 0 ${spacingHorizontalM} ${spacingHorizontalM};
   }
-  :host .secondary-panel-today {
-    color: ${colorNeutralForeground2Selected};
-    background: ${colorBrandBackgroundInvertedSelected};
+  :host .first-transition-row-animated {
+    animation: FadeOut, SlideUpOut20, transitionRowDisappearance;
+    animation-duration: 0.367s;
+    animation-timing-function: cubic-bezier(0.1, 0.9, 0.2, 1);
   }
-  :host .secondary-panel-selected {
-    color: ${colorNeutralForeground2Selected};
-    background: ${colorBrandBackgroundInvertedSelected};
+  :host .last-transition-row-animated {
+    animation: FadeOut, SlideDownOut20, transitionRowDisappearance;
+    animation-duration: 0.367s;
```

Comment:
Same as other animation token comment..

```suggestion
    animation-duration: ${durationUltraFast};
```